### PR TITLE
Gitea: Allow User Login with Token

### LINKED ISSUE #195: 2fa Gitea
Should this work with 2fa and gitea? When I try to use my gitea account I get the following error

time="2021-05-03T16:00:47Z" level=error msg="cannot authenticate user. basic auth required"

### PR DESCRIPTION
**Pull request by 6543**
close  #195

### CODE REVIEW

File: remote/gitea/gitea.go [hunk 1]
```diff
@@ -139,8 +139,8 @@ func (c *client) Login(res http.ResponseWriter, req *http.Request) (*model.User,
 		return nil, nil
 	}
 
-	// Create Client with Basic Auth
-	client, err := c.newClientBasicAuth(username, password)
+	// Create Client with Basic Auth or Token
```
Comment (lunny) [inline remote/gitea/gitea.go:142]: How about move if there is a password here?

Comment (6543) [reply remote/gitea/gitea.go:142]: where? should I put a `move`? 

sorry - I think you have to explain your idea :/

Comment (6543) [reply remote/gitea/gitea.go:142]: where? should I put a `move`? 

sorry - I think you have to explain your idea :/ @lunny

**Comment by techknowlogick** [comment]
Rather than logging in with a password, the alternative of logging in with oauth is better.

**Comment by techknowlogick** [comment]
Now that OAuth support has been added, this could probably be closed.

**Pull request closed without merging**